From stable+bounces-76111-greg=kroah.com@vger.kernel.org Fri Sep 13 22:03:33 2024
From: Samasth Norway Ananda <samasth.norway.ananda@oracle.com>
Date: Fri, 13 Sep 2024 13:02:40 -0700
Subject: selftests/kcmp: remove call to ksft_set_plan()
To: stable@vger.kernel.org
Cc: gautammenghani201@gmail.com, skhan@linuxfoundation.org, usama.anjum@collabora.com, saeed.mirzamohammadi@oracle.com, samasth.norway.ananda@oracle.com
Message-ID: <20240913200249.4060165-3-samasth.norway.ananda@oracle.com>

From: Samasth Norway Ananda <samasth.norway.ananda@oracle.com>

The function definition for ksft_set_plan() is not present in linux-4.19.y.
kcmp_test selftest fails to compile because of this.

Fixes: 32b0469d13eb ("selftests/kcmp: Make the test output consistent and clear")
Signed-off-by: Samasth Norway Ananda <samasth.norway.ananda@oracle.com>
Acked-by: Shuah Khan <skhan@linuxfoundation.org>
Reviewed-by: Saeed Mirzamohammadi <saeed.mirzamohammadi@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 tools/testing/selftests/kcmp/kcmp_test.c |    1 -
 1 file changed, 1 deletion(-)

--- a/tools/testing/selftests/kcmp/kcmp_test.c
+++ b/tools/testing/selftests/kcmp/kcmp_test.c
@@ -89,7 +89,6 @@ int main(int argc, char **argv)
 		int ret;
 
 		ksft_print_header();
-		ksft_set_plan(3);
 
 		fd2 = open(kpath, O_RDWR);
 		if (fd2 < 0) {
